packages/launchpad/cypress/e2e/project-setup.cy.ts

```diff
@@ -347,92 +333,7 @@ describe('Launchpad: Setup Project', () => {
         verifyScaffoldedFiles('e2e')
       })
 
-      it('can setup e2e testing for a project selecting TS when CT is configured and config file is JS', () => {
```

Comment:
You cannot do this anymore, you are either using JS or TS - this feature never made any sense in the first place.